From: Alberto Garcia <berto@igalia.com>
Description: Enable THREADS_PREFER_PTHREAD_FLAG
 This fixes a FTBFS in riscv64
Bug: https://bugs.webkit.org/show_bug.cgi?id=182622
Bug-Debian: https://bugs.debian.org/895969
Origin: https://trac.webkit.org/changeset/231843
Index: webkitgtk/Source/cmake/OptionsGTK.cmake
===================================================================

diff --git Source/cmake/OptionsGTK.cmake Source/cmake/OptionsGTK.cmake
index 9e4db3a25b377177e334ce321c219574aa7a9d66..3985114d054723b22c58dac699ccef8e95975612 100644
--- Source/cmake/OptionsGTK.cmake
+++ Source/cmake/OptionsGTK.cmake
@@ -5,6 +5,9 @@ WEBKIT_OPTION_BEGIN()
 
 SET_PROJECT_VERSION(2 44 3)
 
+# no idea if this is still required; moved up to untangle this change from changes to dep. versions
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+
 # This is required because we use the DEPFILE argument to add_custom_command().
 # Remove after upgrading cmake_minimum_required() to 3.20.
 if (${CMAKE_VERSION} VERSION_LESS "3.20" AND NOT ${CMAKE_GENERATOR} STREQUAL "Ninja")

Index: webkitgtk/Source/cmake/OptionsJSCOnly.cmake
===================================================================
--- Source/cmake/OptionsJSCOnly.cmake
+++ Source/cmake/OptionsJSCOnly.cmake
@@ -1,3 +1,4 @@
+set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
 if (MSVC)
